### PR TITLE
Cleanup dockerfiles

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
- cleanup copyright headers
- fix version of alpine (replace latest by 3.4)
- use of name "entrypoint.sh" for entrypoint scripts

### CODE REVIEW

**Comment by benoitf** [comment]
@riuvshin ?
in particular this commit https://github.com/codenvy/codenvy/pull/1196/commits/5ddfe849228a4399bacf98d4e6127d6a6fcf770f based on comment of last week 
https://github.com/codenvy/codenvy/pull/1188#issuecomment-261662827

File: dockerfiles/cli/version/latest/images [hunk 1]
```diff
@@ -1,10 +1,10 @@
-IMAGE_SOCAT=codenvy/socat:1.7.3.1
-IMAGE_HAPROXY=codenvy/haproxy:1.5.18
-IMAGE_PUPPET=codenvy/puppet:4.6.2
+IMAGE_CODENVY=codenvy/codenvy:5.0.0-M7
+IMAGE_HAPROXY=codenvy/haproxy:5.0.0-M7
+IMAGE_INIT=codenvy/init:5.0.0-M7
```
Comment (riuvshin) [inline dockerfiles/cli/version/latest/images]: latest should use tags `latest` same as nightly

Comment (benoitf) [inline dockerfiles/cli/version/latest/images]: @riuvshin fixed

Comment (benoitf) [inline dockerfiles/cli/version/latest/images]: I will issue a separate PR for this stuff as you disagree only on this part

**Pull request merged**